packages/react-query/src/scripts/cli.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+'use strict'
+
+import { Command } from '@commander-js/extra-typings'
+import { getPackageJson, getSuspensiveReactQueryPackageName, getTanStackReactQueryPackageJson } from './utils/package'
+import { switchVersion } from './utils/switchVersion'
+
+const packageJson = getPackageJson()
+const tanstackPackageJson = getTanStackReactQueryPackageJson()
+const suspensiveReactQueryPackageName = getSuspensiveReactQueryPackageName()
+
+const program = new Command()
+
+program
+  .name(packageJson.name)
+  .description(packageJson.description)
+  .version(packageJson.version, '-v, --version', 'display the current version')
+
+program
+  .command('status')
+  .description('Check the compatibility status of the current version')
+  .action(() => {
+    const tanstackMajorVersion = tanstackPackageJson.version.split('.')[0]
+    const suspensiveReactQueryVersion = suspensiveReactQueryPackageName.split('-')[2]
+
+    console.log(`[@suspensive/react-query]`, `v${packageJson.version}`, `(${suspensiveReactQueryPackageName})`)
+    console.log('[@tanstack/react-query]', `v${tanstackPackageJson.version}`)
+
+    if (suspensiveReactQueryVersion === tanstackMajorVersion) {
+      console.log(`\nThe versions are compatible.`)
+    } else {
+      console.warn(
+        '\nThe version of @suspensive/react-query is not compatible with the current version of @tanstack/react-query.',
+        `\nPlease run 'npx suspensive-react-query switch ${suspensiveReactQueryVersion === '5' ? '4' : '5'}' to switch to the compatible version.`
+      )
+    }
+  })
+
+program
+  .command('switch')
+  .description('Switch to a specific version of @suspensive/react-query')
+  .argument('<version>', 'Switches the version of @suspensive/react-query')
+  .action((version) => {
+    if (version === '4') {
+      switchVersion(4)
+    } else if (version === '5') {
+      switchVersion(5)
+    } else {
+      console.warn('[@suspensive/react-query]', `version v${version} is not supported.`)
+    }
+  })
+
+program
+  .command('fix')
```

Comment:
```shell
Usage: @suspensive/react-query fix [options]

Fix the compatibility issues

Options:
  -h, --help  display help for command
```

I have added a new command `fix`.
This command switches to the appropriate version based on the dependencies.

Reply:
So much better DX! This could replace "suspensive-react-query switch" right?

Reply:
That's right! You can use it like this: `build: "npx srq fix & vite build"`!

Reply:
> That's right! You can use it like this: build: "npx srq fix & vite build"!

Freaking better~!

maybe `"build": "srq fix & vite build"` could work

Then how about naming as `fix` -> `auto`? In my opinion, `fix` could mean that there is something wrong...!
So I want to make it positively

Reply:
"build": "srq auto & vite build

Reply:
Oh, that sounds good! I'll change it right away!

Reply:
hmm fix could be better... I'm really torn between these two `auto`, `fix`

but we use this as experimental feature! so gogo first! as `fix`

Reply:
If `auto` is good for you, you can choose naming! I want to follow your decision

Reply:
Hmm.. I think `fix` fits better since it fixes compatibility issues!

Reply:
👍🚀 I follow your decision